arithmoi.cabal

```diff
@@ -105,9 +105,21 @@ test-suite spec
   type:                 exitcode-stdio-1.0
   hs-source-dirs:       test-suite
   ghc-options:          -Wall
-  main-is:              Spec.hs
+  main-is:              Test.hs
   default-language: Haskell2010
   build-depends:        base
-                        ,hspec
-                        ,arithmoi
+                        , arithmoi
+                        , tasty
+                        , tasty-smallcheck
+                        , tasty-quickcheck
+                        , tasty-hunit
+                        , QuickCheck
+                        , smallcheck
```

Comment:
Just because I worked a lot with `tasty` and never with `hspec`. Also AFAIK `hspec` lacks support of `smallcheck`.